basket/news/migrations/0018_commonvoiceupdate.py

```diff
@@ -31,7 +29,7 @@ class Migration(migrations.Migration):
                         editable=False,
                     ),
                 ),
-                ("data", jsonfield.fields.JSONField(default=dict)),
+                ("data", models.JSONField(default=dict)),
```

Comment:
Had to edit these old ones due to import errors b/c the package was removed. But it should only come into play for new installs or tests.